pkg/kubectl/cmd/label.go

```diff
@@ -136,6 +136,10 @@ func NewCmdLabel(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 // Complete adapts from the command line args and factory to the data required.
 func (o *LabelOptions) Complete(out io.Writer, cmd *cobra.Command, args []string) (err error) {
+	// cannot set --all and --selector at the same time
```

Comment:
use the options object which contains this information to perform the check.

Reply:
@deads2k We can only get the value from the option struct. But we don’t know whether the flags are set/changed through cli. That’s why I define and use a common method for this checking.